lxd/api_cluster.go

```diff
@@ -662,6 +662,19 @@ func clusterPutJoin(d *Daemon, r *http.Request, req api.ClusterPut) response.Res
 			return err
 		}
 
+		// Add the new node to the default cluster group.
+		err = d.cluster.Transaction(func(tx *db.ClusterTx) error {
```

Comment:
Don't we have an earlier cluster transaction that this could be done in (to avoid a complete separate DB call)?